docs/content/configuration/storage.md

```diff
@@ -0,0 +1,223 @@
+---
+title: Configuring Storage
+description: Configuring Storage in Athens
+---
+
+## Storage
+
+The Athens proxy supports many storage types:
+
+1. [Memory](#memory)
+1. [Disk](#disk)
+1. [Mongo](#mongo)
+1. [Google Cloud Storage](#google-cloud-storage)
+1. [AWS S3](#aws-s3)
+1. [Minio](#minio)
+1. [DigitalOcean Spaces](#digitalocean-spaces)
+
+All of them can be configured using `config` file. You need to set proper driver in `StorageType` value or you can set it in environment variable `ATHENS_STORAGE_TYPE` on your server.
+Also in most of drivers you need to provide additional configuration data which will be described below.
+
+## Memory
+
+This storage don't need any specific configuration and it's also used by default in Athens project. It writes all of data into local disk into `tmp` dir.
+
+**This storage type should only be used for development purpose!**
+
+##### config:
+
+    # StorageType sets the type of storage backend the proxy will use.
+    # Env override: ATHENS_STORAGE_TYPE
+    StorageType = "memory"
+    
+## Disk
+
+This storage type is the same as memory but allows to set location of root folder where data should be stored. 
```

Comment:
I would probably not mention memory as that is an implementation detail. The key things is that you can store it on disk given a location.

Reply:
Done. Thanks for help @manugupt1 !